Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -268,6 +268,85 @@
+    sound:
+      path: /Audio/Items/smoke_grenade_prime.ogg
+      params:
+        volume: 15
+  - type: TimedDespawn
+    lifetime: 3
+
+- type: entity
+  id: BulletTearGasGrenade
+  name: tear gase grenade
+  parent: BulletSmokeGrenade
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)